fineract-provider/src/main/java/org/apache/fineract/scheduledjobs/service/ScheduledJobRunnerServiceImpl.java

```diff
@@ -443,43 +466,109 @@ public void updateTrialBalanceDetails() throws JobExecutionException {
                     .append("from acc_gl_journal_entry je WHERE je.transaction_date = ? ")
                     .append("group by je.account_id, je.office_id, je.transaction_date, Date(je.entry_date)");
 
-            final int result = jdbcTemplate.update(sqlBuilder.toString(), new Object[] {
+            final int result = jdbcTemplate.update(sqlBuilder.toString(), new Object[]{
```

Comment:
There are a number of formatting only and lower case to uppercase changes proposed here which make this PR harder to review than it would be if it consistet only of additional new functional code? Would you consider to propose these formatting only and lower case to uppercase changes in a separate PR instead, and undo them from here? Just asking.

Reply:
@vorburger  @avikganguly01  Opened a new PR after reading feedback.
[PR #512 ](https://github.com/apache/fineract/pull/512)